ansible/roles/operator-pipeline/templates/openshift/tasks/github-pipelinerun-summary.yml

```diff
@@ -117,9 +120,15 @@ spec:
           echo "Pipeline logs: $GIST_URL" >> $PR_NAME/comment.md
         fi
 
-        if [ ! -z "$(params.comment_suffix)" ]; then
-          echo "$(params.comment_suffix)" >> $PR_NAME/comment.md
-        fi
+        CONFIG_PATH="$WORKSPACE_BASE_PATH/config.yaml"
+        ORGANIZATION=`cat $CONFIG_PATH | yq -r '.organization'`
+
+        [[ "$ORGANIZATION" == "certified-operators" || "$ORGANIZATION" == "redhat-marketplace" ]] \
```

Comment:
I know this looks fancy to have just a one-liner for the link :D but given this is shell script can you transform it to something more readable? if/else with proper indentation?

Reply:
Also, it might be easier to switch the logic and check if a value is equal to `community-operators`. That way you have just one hardcoded value instead of 2.

Reply:
Sure, I'll replace it